vendor/extensions/reports/app/controllers/refinery/reports/admin/reports_controller.rb

```diff
@@ -6,21 +6,23 @@ class ReportsController < ::Refinery::AdminController
         crudify :'refinery/reports/report'
 
         def create
-          @report = Refinery::Reports::Report.create(report_params)
-          if params[:tag]
-            tags = params[:tag][:tag_ids].each do |tid|
-              Refinery::Taggings::Tagging.create(report_id: @report.id, tag_id: tid.to_i)
-            end
+          @report = Refinery::Reports::Report.new(report_params)
+          params[:tag][:tag_ids].each do |tid|
+            @report.tags << Refinery::Tags::Tag.find(tid)
+          end
+          if @report.save
+            flash[:notice] = "Report was successfully created!"
+            redirect_to reports_admin_reports_path
+          else
+            render action: 'new'
           end
-          @report.save
-          redirect_to reports_admin_reports_path and return
         end
 
         def update
           @report = Refinery::Reports::Report.find(params[:id])
           @report.update(report_params)
           @report.taggings.each {|t| t.delete}
-          tags = params[:tag][:tag_ids].each do |tid|
+          params[:tag][:tag_ids].each do |tid|
```

Comment:
Am I correct in understanding that this change just removes an inline variable declaration that wasn't used elsewhere?

Reply:
Yep, removed dead code!